modules/core/portal-bootstrap/system.packages.extra.bnd

```diff
@@ -223,6 +223,12 @@ Export-Package:\
 	org.apache.commons.logging.impl;version="1.2",\
 	org.apache.commons.validator.routines;version='1.5.1',\
 	org.apache.log4j;version='2.14.0',\
+	org.apache.log4j.bridge;version='2.14.0',\
```

Comment:
@yuhai portal-test is deployed as a module, but my `RollingFileAppenderBuilder` is in petra-log4j which is deployed as a lib in core, which is in the same class loader as log4j jars. What packages does the latter one need?

Reply:
@dantewang, except org.apache.log4j.spi, RollingFileAppenderBuilder needs the other packages. I will remove these.

Reply:
@dantewang, I just push force the change. Only keep portal-test needed.